src/subcommands/upgrade.ts

```diff
@@ -70,7 +70,6 @@ export default async function (rawArgs: Record<string, any>): Promise<void> {
         "--allow-net",
         "--allow-run",
         "--no-check",
-        "--unstable",
```

Comment:
What do you think about retaining `--unstable`? It will come in handy if we want to ship a version that uses an unstable API.